archive_converter.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import email
+from setup_gmail import Message
+
+def generate_email_from_file():
+    raw_email = open("patches/patch6.txt", "r")
```

Comment:
Can we make generate_email_from_file() take the file path as a parameter?

Reply:
yeah that was a thought that crossed my mind after I submitted the PR haha